Firmware/Marlin_main.cpp

```diff
@@ -11150,15 +11149,15 @@ void stop_and_save_print_to_ram(float z_move, float e_move)
     {
         // First unretract (relative extrusion)
         if(!saved_extruder_relative_mode){
-            enquecommand(PSTR("M83"), true);
```

Comment:
hmmmmm. Wasn't this actually broken in the past? It looks to me as if the wrong enqueuecommand function type was used in the past, treating a progmem reference as a ram reference.